templates/long.yaml

```diff
@@ -32,3 +32,13 @@
 
 - name: Test Project - Empty
   type: Project
+
+- name: Test Team
+  type: Team
+  can_public_join: false
+  description: Test team description
+  invitations:
+  - message: Welcome to the Test Team! 
```

Comment:
After attempting to add this to the client, I realize this would cause a nested for loop which is pretty ugly.   Something like this:

```
            for invite in config['invitations']:
                for member in config['members']:
                    user = member.get("principal_id")
                    email = member.get("email")
                    creation_cls.syn.invite_to_team(
                        team=team, user=user, inviteeEmail=email,
                        message=invite['message']
                    )
```

Reply:
I commented on the other PR, but I think this "natural" if we want to represent it that way.  And it's not a big deal to write a nested loop for something this small.  Your loop is still readable!